src/streamlit_extras/__init__.py

```diff
@@ -50,7 +50,11 @@ def extra(
             module.__funcs__ = [func]  # type: ignore
 
         profiling_name = f"{submodule}.{func.__name__}"
-        return _gather_metrics(name=profiling_name, func=func)
+        try:
+            return _gather_metrics(name=profiling_name, func=func)
+        except TypeError:
+            # Don't fail on streamlit==1.13.0, which only expects a callable
```

Comment:
So it won't fail, but won't gather metrics for streamlit==1.13.0 is that right? Fair enough

Reply:
Yeah, I wasn't sure what else to do with it, since it would otherwise have a weird name, so wouldn't be terribly usable to have those metrics.